### ISSUE TITLE
Secondary screen not showing.

### ISSUE THREAD

**Issue by chixxi** [741g/android-emulator-m1-preview#4]
To be honest, I don't know if I am doing something wrong here or if this is a bug.

I am trying to setup a second screen, mainly cause the resolution of the primary screen can not be customized. The second screen is never shown in the emulator, independant from using a custom or preset resolution.

**Comment by 741g** [comment]
To directly address your original issue, the primary resolution can be set via editing `/Applications/Android Emulator.app/Contents/MacOS/aosp-master-arm64-v8a/config.ini`'s `hw.lcd.width` and `hw.lcd.height` keys.

**Comment by 741g** [comment]
On the matter of the multi display UI not working, it can be due to not having the multi display specific service installed/running in the guest.

**Comment by chixxi** [comment]
> To directly address your original issue, the primary resolution can be set via editing `/Applications/Android Emulator.app/Contents/MacOS/aosp-master-arm64-v8a/config.ini`'s `hw.lcd.width` and `hw.lcd.height` keys.

This is absolutely perfect, thanks so much for this info! For my personal view this issue can be closed.

**Comment by chixxi** [comment]
> On the matter of the multi display UI not working, it can be due to not having the multi display specific service installed/running in the guest.

I installed this for all users. Is there any way to verify this? If you feel like I can help solve the issue let me know, or feel free to close as my personal issue is solved.

**Comment by chixxi** [comment]
What are the conditions to satisfy in regards to DPI? Seems like if I go below 320 the emulator is no longer launching.

**Comment by 741g** [comment]
> > On the matter of the multi display UI not working, it can be due to not having the multi display specific service installed/running in the guest.
> 
> I installed this for all users. Is there any way to verify this? If you feel like I can help solve the issue let me know, or feel free to close as my personal issue is solved.

Interesting, how did you install it? I'd like to replicate it locally.

**Comment by chixxi** [comment]
Hm, I guess that was a wrong claim by me. I simply downloaded the *.dmg file, opened it and pulled it over to "Applications". I manuall set the path to ADB (/Users/username/Library/Android/sdk/platform-tools/adb).

I did not use the ZIP in combination with the Android SDK.

I did re-install V2 using the same procedure, I am now getting the second screen:
<img width="1723" alt="Screenshot 2020-12-11 at 10 08 42" src="https://user-images.githubusercontent.com/1770007/101884411-ef40c800-3b98-11eb-90c3-89d57f97edc3.png">
